realtime/server.js

```diff
@@ -21,14 +21,14 @@ var people = {};
 var usernames = {};
 var messages_typing = {};
 
-function getOptions(form, target, method) {
+function getOptions(form, params, method) {
```

Comment:
`target` was a better name. Alternatives could be `url` or `endpoint`. HTTP parameters refer to the portion after the `?` usually.